src/App.module.css

```diff
@@ -15,3 +15,9 @@
     display: block;
   }
 }
+
+html, body {
+  overflow-x: hidden;
+  width: 100%;
+  max-width: 100vw;
```

Comment:
I swear we had this set somewhere else to prevent it which seemed to be working on FF.